src/tokenomics.md

```diff
@@ -69,8 +69,6 @@ through [SORA governance](/sora-governance.md).
   token bonding curve.
 - The [PUSSY](/pussy.md) token is a dedicated meme cryptocurrency in the SORA economy aimed
   at funding and rewarding builders.
```

Comment:
_"aimed at funding and rewarding builders."_ - I think it's no longer the case

Reply:
Edited.

Reply:
Done